tests/integration_tests/test_cli.py

```diff
@@ -319,53 +321,9 @@ def test_collect_summary_activations_weights_cli(csv_filename):
                                         )
         stdout = completed_process.stdout.decode('utf-8')
 
-        # parse output of collect_summary to find tensor names to use
-        # in the collect_wights and collect_activations.
-        # This part of test is sensitive to output format of collect_summary
-        # search for substring with layer names
-        layers = re.search(
-            "Layers(\w|\d|\:|\/|\n)*Weights",
-            stdout
-        )
-        substring = stdout[layers.start(): layers.end()]
-
-        # extract layer names
-        layers_list = []
-        with StringIO(substring) as f:
-            for _, line in enumerate(f):
-                if not (line[:6] == 'Layers' or line[:6] == 'Weight') and len(
-                        line) > 1:
-                    layers_list.append(line[:-1])
-
-        # search for substring with weights names
-        weights = re.search(
-            "Weights(\w|\d|\:|\/|\n)*",
-            stdout
-        )
-        substring = stdout[weights.start(): weights.end()]
-
-        # extract weights names
-        weights_list = []
-        with StringIO(substring) as f:
-            for _, line in enumerate(f):
-                if (not (line[:6] == 'Layers' or line[:6] == 'Weight')
-                        and len(line) > 1):
-                    weights_list.append(line[:-1])
-
-        # collect activations
-        _run_ludwig('collect_activations',
-                    dataset=dataset_filename,
-                    model_path=os.path.join(tmpdir, 'experiment_run', 'model'),
-                    layers=' '.join(layers_list),
-                    output_directory=tmpdir
-                    )
-
-        # collect weights
-        _run_ludwig('collect_weights',
-                    model_path=os.path.join(tmpdir, 'experiment_run', 'model'),
-                    tensors=' '.join(weights_list),
-                    output_directory=tmpdir
-                    )
+        # TODO(justin): Check that more summary information was printed using torchsummary.
```

Comment:
Try to understand why the previous code was removed

Reply:
The context for this change is in the PR description.

Basically, I don't think there's much value from having this test manually parsing stdout, since the actual content of what will get printed will already be tested in `test_collect`.